packages/teleport/src/lib/tdp/codec.ts

```diff
@@ -63,8 +79,8 @@ export type ClipboardData = {
 
 // | message type (10) | mfa_type byte | message_length uint32 | json []byte
 export type MfaJson = {
-  // TODO(isaiah) make this a type that ensures it's the same as MessageTypeEnum.U2F_CHALLENGE and MessageTypeEnum.WEBAUTHN_CHALLENGE
-  mfaType: 'u' | 'n';
```

Comment:
Technically `'u'` is still a part of the [TDP spec](https://github.com/gravitational/teleport/blob/master/rfd/0037-desktop-access-protocol.md#10---mfa), so we should leave this as is until that's changed.

Reply:
reverted the file